validator/testcases/javascript/call_definitions.py

```diff
@@ -167,6 +167,26 @@ def spellcheck_updatecurrentdictionary(wrapper, arguments, traverser):
             tier=5)
 
 
+def webbrowserpersist(wrapper, arguments, traverser):
+    """
+    Most nsIWebBrowserPersist should no longer be used, in favor of the new
+    Downloads.jsm interfaces.
+    """
+    traverser.err.warning(
+        err_id=("testcases_javascript_call_definititions",
+         "webbrowserpersist"),
```

Comment:
Yeah. I copied that from from the test below and decided to just err on the side of consistency, but I'll just change them both. (flake8 incidentally complains so much about this file that looking at the output is barely even useful)